src/modules/core/components/IndexModal/IndexModalItem.tsx

```diff
@@ -2,37 +2,45 @@ import React from 'react';
 import { FormattedMessage, defineMessages } from 'react-intl';
 import Icon from '~core/Icon';
 import Paragraph from '~core/Paragraph';
-import styles from './ColonyActionsDialogItem.css';
+import styles from './IndexModalItem.css';
 
 interface Props {
   title: object;
   description: object;
-  disabled?: boolean;
+  comingSoon?: boolean;
   icon: string;
+  onClick?: () => void;
 }
 
 const MSG = defineMessages({
   coming: {
-    id: 'dashboard.ColonyActionsDialog.ColonyActionsDialogItem.coming',
+    id: 'core.IndexModal.IndexModalItem.coming',
     defaultMessage: 'Coming soon',
   },
 });
 
-const ColonyActionsDialogItem = ({
+const IndexModalItem = ({
   title,
   description,
-  disabled,
   icon,
+  onClick,
+  comingSoon,
 }: Props) => {
   return (
-    <div className={`${disabled ? styles.disabled : styles.content}`}>
+    <div
+      className={`${comingSoon ? styles.disabled : styles.content}`}
```

Comment:
This can be achieved more cleanly with `getMainClasses`, but it's not major. If you don't have time, just leave it like this.